web/src/server_events_dispatch.js

```diff
@@ -165,7 +165,12 @@ export function dispatch_normal_event(event) {
             activity_ui.update_presence_info(event.user_id, event.presence, event.server_timestamp);
             break;
 
-        case "restart": {
+        case "restart":
+            page_params.zulip_version = event.zulip_version;
+            page_params.zulip_merge_base = event.zulip_merge_base;
+            break;
```

Comment:
We might want to have the web app track separate `zulip_server_{version|merge_base}` and `zulip_web_app_{version|merge_base}` variables; we can always do such a separation later, but for some purposes, likely including a variable in `sentry`, we do want the version of the web app.

Reply:
We already track the version -- `ZULIP_VERSION` is a constant built into the webpack'd source (a8181152b7f9594bfdb4d9fdce83f1b27841f498, 19691b170bcb59bfad122e811971ca6b168e5855).

Reply:
Ahh, OK, and this is used in Sentry, so actually this is fixing a bug in the web app that it would have a stale value of `server_version` until reloaded, OK.